vendor/miniaudio/common.odin

```diff
@@ -13,13 +13,15 @@ when ODIN_OS == .Windows {
 handle :: distinct rawptr
 
 
-/* SIMD alignment in bytes. Currently set to 64 bytes in preparation for future AVX-512 optimizations. */
-SIMD_ALIGNMENT :: 64
-
-LOG_LEVEL_DEBUG   :: 4
-LOG_LEVEL_INFO    :: 3
-LOG_LEVEL_WARNING :: 2
-LOG_LEVEL_ERROR   :: 1
+/* SIMD alignment in bytes. Currently set to 32 bytes in preparation for future AVX optimizations. */
+SIMD_ALIGNMENT :: 32
+
+log_level :: enum c.int {
+	LOG_LEVEL_DEBUG   = 4,
+	LOG_LEVEL_INFO    = 3,
+	LOG_LEVEL_WARNING = 2,
+	LOG_LEVEL_ERROR   = 1,	
+}
```

Comment:
Why has this become an enum rather than the global constants?

Reply:
Just to keep in line with the change from `#define` constants (in `v0.10.x`) to `enum` (in `v0.11.x`) for the same in the actual miniaudio source header.